js/views/RegistrationPages.js

```diff
@@ -123,7 +123,8 @@ const RegistrationPages = React.createClass ({
    */
   beforePageChange(currentPage, nextPage) {
     const shouldContinue = this.validatePage();
-    if (shouldContinue || nextPage < currentPage) {
+    if (shouldContinue || nextPage >= 0 && nextPage < currentPage) {
```

Comment:
Scenario:
Page gets set to -1 index, but the swiper prevents it from moving backwards, so the view thinks it's on -1. The validation never works for -1 index, so it allows the user to skip validation for the first page.

By not letting that index get set to -1, it prevents the first-page skip from happening.